attestation-ca/src/lib.rs

```diff
@@ -69,18 +75,23 @@ impl TryFrom<SerialisableAttestationCa> for AttestationCa {
 
     fn try_from(data: SerialisableAttestationCa) -> Result<Self, Self::Error> {
         Ok(AttestationCa {
-            ca: x509::X509::from_der(data.ca.as_slice())?,
+            ca_openssl: ox509::X509::from_der(data.ca.as_slice())?,
+            ca: x509::Certificate::from_der(data.ca.as_slice()).unwrap(),
             aaguids: data.aaguids,
             blanket_allow: data.blanket_allow,
         })
     }
 }
 
 impl AttestationCa {
-    pub fn ca(&self) -> &x509::X509 {
+    pub fn ca(&self) -> &x509::Certificate {
         &self.ca
     }
 
+    pub fn ca_openssl(&self) -> &ox509::X509 {
```

Comment:
Should this interface be marked as deprecated?

Reply:
It will be naturally killed when I remove openssl since it's returning an openssl type.